skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -4,19 +4,19 @@ import '../scss/navbar.css';
 import 'font-awesome/css/font-awesome.min.css';
 import '../bootstrap/NavbarMenuStyle.css';
 import history from './history';
-import profileRed from '../components/navbar-icons/profile-red.png';
-import profileGray from '../components/navbar-icons/profile-grey.png';
-import docsRed from '../components/navbar-icons/docs-red.png';
-import docsGray from '../components/navbar-icons/docs-grey.png';
-import createRed from '../components/navbar-icons/create-red.png';
-import createGray from '../components/navbar-icons/create-grey.png';
-import leaderboardRed from '../components/navbar-icons/leaderboard-red.png';
-import leaderboardGray from '../components/navbar-icons/leaderboard-grey.png';
-import notificationsRed from '../components/navbar-icons/notifications-red.png';
-import notificationsGray from '../components/navbar-icons/notifications-grey.png';
-import scorecaptureRed from '../components/navbar-icons/add-score-red.png';
-import scorecaptureGray from '../components/navbar-icons/add-score-grey.png';
-import moreGray from '../components/navbar-icons/more-grey.png';
+import profilered from '../components/navbar-icons/profile-red.png';
```

Comment:
This was correct

Reply:
if you create an icons constant object then you can have one import in here like 
import NAV_BAR_ICONS from 'constantsFile'
then just  `src={ NAV_BAR_ICONS.PROFILE_GRAY }`
just use real names, no docsgray more like DOCUMENT_GRAY

Reply:
okay done